_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,375 @@
+will then happen in Flink 1.16.
+
+
+### Changelog state backend
+
+From Flink 1.15, we have the MVP feature of 
+[changelog state](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/state_backends/#enabling-changelog)
+backend, which aims at 
+making checkpoint intervals shorter with following advantages:
+
+1. Less work on recovery: The more frequently a checkpoint is taken, the fewer events 
+   need to be re-processed after recovery.
+2. Lower latency for transactional sinks: Transactional sinks commit on checkpoints, 
+   so faster checkpoints mean more frequent commits.
+3. More predictable checkpoint intervals: Currently the length of the checkpoint mainly
+   depends on the size of the artifacts that need to be persisted on the checkpoint 
+   storage, by keeping the size small it becomes more predictable.
+
+This work introduced in Flink 1.15 helps achieve the above advantages by continuously 
+persisting state changes on a non-volatile storage, while performing materialization 
+in the background.
+
+
+### Adaptive batch scheduler
+
+In 1.15, we introduced a new scheduler to Apache Flink: the 
+[Adaptive Batch Scheduler](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/deployment/elastic_scaling/#adaptive-batch-scheduler). 
+The new scheduler can automatically decide parallelisms of job vertices for batch jobs, 
+according to the size of data volume each vertex needs to process.
+
+
+### Other things worth mentioning
+
+There have been improvements to the application mode. Jobs that should take a savepoint 
+after they are completed, can now guarantee they do so 
+([see execution.shutdown-on-application-finish](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/deployment/config/#execution-shutdown-on-application-finish)). 
+Recovery and clean up of jobs running in application mode have been improved. The local 
+state can be persisted in the working directory to improve the experience of recovering 
+from local storage.
+
+
+## Unification of stream and batch processing - once more
+
+In the latest release the unification of stream and batch processing is the main topic. 
+This time some new efforts have been picked up, others have been continued.
+
+
+### Final checkpoints
+
+In 1.14 final checkpoints were added as a feature that had to be enabled manually. 
+Since the last release users have been providing feedback to the community that has 
+now been incorporated, giving us the confidence to enable it by default. For more 
+information and how to disable this feature please refer to the 
+[documentation](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/datastream/fault-tolerance/checkpointing/#checkpointing-with-parts-of-the-graph-finished).
+This change in configuration can prolong the shutting down sequence of bounded 
+streaming jobs, as jobs have to wait for a final checkpoint before being allowed to 
+finish.
```

Comment:
```suggestion
In Flink 1.14, final checkpoints were added as a feature that had to be enabled manually. 
Since the last release, we listened to user feedback and decided to enable it by default. For more 
information and how to disable this feature, please refer to the 
[documentation](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/datastream/fault-tolerance/checkpointing/#checkpointing-with-parts-of-the-graph-finished).
This change in configuration can prolong the shutting down sequence of bounded 
streaming jobs because jobs have to wait for a final checkpoint before being allowed to 
finish.
```